evap/evaluation/models.py

```diff
@@ -428,16 +447,22 @@ def save(self, *args, **kw):
         assert self.vote_end_date >= self.vote_start_datetime.date()
 
         if hasattr(self, 'state_change_source'):
-            state_changed_to = lambda self, state_set: self.state_change_source not in state_set and self.state in state_set
-            state_changed_from = lambda self, state_set: self.state_change_source in state_set and self.state not in state_set
+            state_changed_to = (
+                lambda self, state_set: self.state_change_source not in state_set and self.state in state_set
+            )
+            state_changed_from = (
+                lambda self, state_set: self.state_change_source in state_set and self.state not in state_set
+            )
 
             # It's clear that results.models will need to reference evaluation.models' classes in ForeignKeys.
             # However, this method only makes sense as a method of Evaluation. Thus, we can't get rid of these imports
             # pylint: disable=import-outside-toplevel
             from evap.results.tools import STATES_WITH_RESULTS_CACHING, STATES_WITH_RESULT_TEMPLATE_CACHING
 
             if (
-                state_changed_to(self, STATES_WITH_RESULTS_CACHING) or self.state_change_source == 'evaluated' and self.state == 'reviewed'
+                state_changed_to(self, STATES_WITH_RESULTS_CACHING)
+                or self.state_change_source == 'evaluated'
+                and self.state == 'reviewed'
```

Comment:
At some points, we have `or` and `and` being put into new lines where I feel it obscures operator precedence for the reader.

(Also, ok with that, just adding it to the discussion)